include/boost/beast/http/parser.hpp

```diff
@@ -84,21 +84,17 @@ class parser
     /// Destructor
     ~parser() = default;
 
-    /// Constructor
-    parser();
-
-    /// Constructor
+    /// Constructor (disallowed)
     parser(parser const&) = delete;
 
-    /// Assignment
+    /// Assignment (disallowed)
     parser& operator=(parser const&) = delete;
 
-    /** Constructor
+    /// Constructor (disallowed)
+    parser(parser&& other) = delete;
```

Comment:
No need for `parser(parser&& other) = delete;` since you explicitly delete the copy constructor. e.g. You're not deleting the move-assignment operator since you're letting the explicit deletion of the copy-assignment operator imply that.

Reply:
This is so it shows up in the generated documentation